DigitalLearningSolutions.Web/Views/LearningPortal/Current/RemoveCurrentActionPlanResourceConfirmation.cshtml

```diff
@@ -0,0 +1,27 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.Current
+@model RemoveActionPlanResourceViewModel
+
+@{
+  ViewData["Title"] = $"Learning Portal - {Model.Name} - Remove - Confirmation";

```

Comment:
Is there a reason the title is in this format?

Reply:
I've kept this the same format as the Remove Course page (initially copied the page content and then modified it) to keep the titles. Do you reckon its worth changing?

Reply:
Not if it's consistent with other pages within the Learning Portal.